app/helpers.php

```diff
@@ -602,8 +602,18 @@ function fast_imagesize($url)
         $data = curl_exec($curl);
         curl_close($curl);
 
-        return read_image_properties_from_string($data);
+        $result = read_image_properties_from_string($data);
```

Comment:
returning null allows null-coalescing stuff in case default is available `myfunc() ?? ['default']`